src/cards/stats-card.js

```diff
@@ -185,6 +188,7 @@ const renderStatsCard = (stats = {}, options = { hide: [] }) => {
         shiftValuePos:
           (!include_all_commits ? 50 : 35) + (isLongLocale ? 50 : 0),
         bold: text_bold,
+        format_stats,
```

Comment:
Please add tests to the https://github.com/anuraghazra/github-readme-stats/blob/master/tests/renderStatsCard.test.js file.